misc/fts.c

```diff
@@ -31,8 +31,13 @@
 static char sccsid[] = "@(#)fts.c	8.6 (Berkeley) 8/14/94";
 #endif /* LIBC_SCCS and not lint */
 
+/* Needed for _DARWIN_FEATURE_ONLY_64_BIT_INODE check below. */
+#if defined(__APPLE__)
+#include <sys/cdefs.h>
+#endif
```

Comment:
We can avoid this `#include` by checking for `__aarch64__` instead of `_DARWIN_FEATURE_ONLY_64_BIT_INODE`, but an architecture check here seems a bit hacky. I'm willing to change it though if desired.